src/main/impl/Config.cpp

```diff
@@ -0,0 +1,422 @@
+#include <boost/algorithm/string.hpp>
+
+#include <main/Config.h>
+
+#include <fstream>
+#include <iostream>
+
+boost::json::object
+parseConfig(std::string const& filename)
+{
+    try
+    {
+        std::ifstream in(filename, std::ios::in | std::ios::binary);
+        if (in)
+        {
+            std::stringstream contents;
+            contents << in.rdbuf();
+            in.close();
+            std::cout << contents.str() << std::endl;
+            boost::json::value value = boost::json::parse(contents.str());
+            return value.as_object();
+        }
+    }
+    catch (std::exception const& e)
+    {
+        std::cout << e.what() << std::endl;
```

Comment:
Should be logged, not cout

Reply:
Logging is not initialized at this point

Reply:
So we can't log yet. 

Logging is initialized based on the values in the configuration.